### DISCUSSION TITLE
Ikea TRADFRIbulbE14WSglobeopal470lm Unsupported

### DISCUSSION THREAD

**Discussion by jdvuyk** [Koenkk/zigbee2mqtt#12260] [Devices]
Hi.
I got one of these bulb from Ikea.  Its a currently supported bulb (https://www.zigbee2mqtt.io/devices/LED2002G5.html) but I cannot seem to get this to pair in a supported way.  Im not really sure where to start. I have a large number of other Ikea devices (switches, button, sensors) that work fine.  just not this one.

Z2MQQT version: 1.18.1
Generic USB CC2538 Stick.  Running second to latest firmware (20201010).  The latest zigbee firmware was only released very recently and doesn't appear to address this in any way.

I have done extensive reading/searching.  Fairly new to Z2M, so any suggestions appreciated.

![image](https://user-images.githubusercontent.com/15186498/164971193-7364fa9b-4112-407b-a147-693a3e6f5dfe.png)

**Comment by Koenkk** [comment]
You are using a very old z2m version.

If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Reply by jdvuyk** [reply]
OK.  that totally fixed my issue.  Thank for pointing me in the right direction.
I had no idea there was a repo migration at some point so would never have figured this out on my own.
As a suggestion, could you consider putting a 'version' tag in the supported device documentation?  So "Supported Since: 1.19.1".  This would have lead me in the direction of figuring out why I had the wrong version.  
Thanks very much!
